### PR TITLE
Hostapd initialization improvement.

### PR DESCRIPTION
**Pull request by mbajorski**
Moved home-ap startup from built-in command to hostapd_cli command.
There are stability issues when wifi0 and wifi1 configs are added
directly to main hostapd process.

Align IP configuration for bhaul and onboard network

### CODE REVIEW

**Comment by jamcoole** [comment]
the  your changes to how you are starting hostapd
we currently check the output of ps to verify that after the image has booted that hostapd0 and hostapd1 are running, e.g. current build
`
4040 ?        Ss     0:01 /usr/sbin/hostapd -g /var/run/hostapd/global -B -P /var/run/hostapd-global.pid /nvram/hostapd0.conf /nvram/hostapd1.conf`
```

hostapd.service - Hostapd IEEE 802.11n AP, IEEE 802.1X/WPA/WPA2/EAP/RADIUS Authenticator
     Loaded: loaded (/lib/systemd/system/hostapd.service; enabled; vendor preset: enabled)
     Active: active (running) since Mon 2021-06-28 11:29:03 UTC; 2min 51s ago
    Process: 3629 ExecStartPre=/bin/sh -c (/lib/rdk/hostapd-init.sh) (code=exited, status=0/SUCCESS)
    Process: 4040 ExecStart=/usr/sbin/hostapd -g /var/run/hostapd/global -B -P /var/run/hostapd-global.pid /nvram/hostapd0.conf 
```
with PR 321:
```
root      4024     1  0 11:57 ?        00:00:07 /usr/sbin/hostapd -g /var/run/hostapd/global -B -P /var/run/hostapd-global.pid
root@TurrisOmnia-GW:~# systemctl -l status hostapd
``````

● hostapd.service - Hostapd IEEE 802.11n AP, IEEE 802.1X/WPA/WPA2/EAP/RADIUS Authenticator
     Loaded: loaded (/lib/systemd/system/hostapd.service; enabled; vendor preset: enabled)
     Active: active (running) since Mon 2021-06-28 11:57:34 UTC; 14min ago
    Process: 3606 ExecStartPre=/bin/sh -c (/lib/rdk/hostapd-init.sh) (code=exited, status=0/SUCCESS)
    Process: 4004 ExecStart=/usr/sbin/hostapd -g /var/run/hostapd/global -B -P /var/run/hostapd-global.pid (code=exited, status=0/SUCCESS)
    Process: 4025 ExecStartPost=/usr/sbin/hostapd_cli -i global raw ADD bss_config=phy0:/nvram/hostapd0.conf (code=exited, status=0/SUCCESS)
    Process: 4128 ExecStartPost=/usr/sbin/hostapd_cli -i global raw ADD bss_config=phy1:/nvram/hostapd1.conf (code=exited, status=0/SUCCESS)
   Main PID: 4024 (hostapd)
     CGroup: /system.slice/hostapd.service
             └─4024 /usr/sbin/hostapd -g /var/run/hostapd/global -B -P /var/run/hostapd-global.pid
```

so if the PR is merged I will just have to remove the individual hostapd0/1 checks and just leave the hostapd service check

File: recipes-connectivity/hostapd/files/hostapd.service [hunk 1]
```diff
@@ -7,7 +7,9 @@ StartLimitIntervalSec=120
 Type=forking
 ExecStartPre=/bin/sh -c '(/lib/rdk/hostapd-init.sh)'
 PIDFile=/var/run/hostapd-global.pid
-ExecStart=/usr/sbin/hostapd -g /var/run/hostapd/global -B -P /var/run/hostapd-global.pid /nvram/hostapd0.conf /nvram/hostapd1.conf
+ExecStart=/usr/sbin/hostapd -g /var/run/hostapd/global -B -P /var/run/hostapd-global.pid
+ExecStartPost=hostapd_cli -i global raw ADD bss_config=phy0:/nvram/hostapd0.conf
```
Comment (manigandanj) [inline recipes-connectivity/hostapd/files/hostapd.service:11]: should we have wifi0 instead of phy0?
should we have wifi1 instead of phy1?

**Review by manigandanj — CHANGES_REQUESTED** [review]

File: recipes-connectivity/hostapd/files/hostapd.service (hunk 1 above)
Comment (mbajorski) [reply recipes-connectivity/hostapd/files/hostapd.service:11]: This is just phy number.
That name is used only for handling group of interfaces on one radio in hostapd internals.
It is not used anywhere else.

Comment (manigandanj) [reply recipes-connectivity/hostapd/files/hostapd.service:11]: AFAIK, phy0 represents 5GHz radio and phy1 represents 2.4GHz radio.
correct me if i am wrong.

**Review by manigandanj — CHANGES_REQUESTED** [review]

**Review by manigandanj — APPROVED** [review]

**Review by rajkumar154 — APPROVED** [review]

**Pull request merged**